tests/integration/golang/helpers/env.go

```diff
@@ -0,0 +1,19 @@
+package helpers
+
+import "os"
+
+func GetDatabaseUri() string {
+	uri, ok := os.LookupEnv("FML_DATABASE_URI")
+	if ok {
+		return uri
+	}
+	return "sqlite://fasttrackml.db"
```

Comment:
fallback? probably much better to return an error or nothing(which could lead test just fail) so we always will see the problem. in case of fallback problem could be just hidden but w can try.

Reply:
I think it's a better experience when you have good defaults. In this particular case, it means that whether inside of outside of a devcontainer, you can just run `make run` in a terminal and `make test-go-integration` in another and it will just work. If you want to run against a different DB, you set the `FML_DATABASE_URI` to the same value in both terminals, and off you go!

Reply:
ok. lets try.